src/healthcheck_http.cpp

```diff
@@ -256,16 +267,21 @@ void Healthcheck_http::event_callback(struct bufferevent *bev, short events,
 
   message = fmt::sprintf("HTTP code %s", statusline);
 
+  for (auto drain_code : hc->drain_codes)
```

Comment:
Not sure, why not `int` here? and for ok_codes as well?

Reply:
Usage of strings is documented in line 58 when they are loaded from json.